packages/payload/src/bin/generateTypes.ts

```diff
@@ -50,10 +27,8 @@ export async function generateTypes(): Promise<void> {
 
   const jsonSchema = configToJSONSchema(payload.config, payload.db.defaultIDType)
 
-  const collectionDeclaration = generateEntityDeclarations(config.collections, 'collections')
-  const globalDeclaration = generateEntityDeclarations(config.globals, 'globals')
-  const declare = `declare module 'payload' {\n  export interface GeneratedTypes {\n${collectionDeclaration}\n${globalDeclaration}\n  }\n}`
-
+  const declare = `declare module "payload" {\n  export interface GeneratedTypes extends Config {} \n}`
```

Comment:
Small nitpick: remove the space between `{}` and `\n`. This equates to trimming whitespace on the generated file.